task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -32,6 +34,7 @@ spec:
         relImgs=$(cat "$(workspaces.workspace.path)/hacbs/fbc-validation/confdir/catalog.yaml" | yq -r '.relatedImages[].image' | sed 's/---//')
         if [ $? -ne 0 ]; then
           echo "Could not get related images, make sure catalog.yaml exists in FBC fragment image and has valid yaml format."
+          note="Task $(context.task.name) meets error: Could not get related images, make sure catalog.yaml exists in FBC fragment image and has valid yaml format."
```

Comment:
suggested change: note="Task $(context.task.name) Failed: Could not fetch related images. Make sure you have `catalog.yaml` formatted correctly in your FBC fragment image."

Reply:
Updated. Thanks.